src/epk3.c

```diff
@@ -263,11 +272,13 @@ void extractEPK3(MFILE *epk, FILE_TYPE_T epkType, config_opts_t *config_opts){
 			segNo < segmentInfo.segmentCount;
 			segNo++, pak++, i++
 		){
-			dataPtr += SIGNATURE_SIZE; //skip segment signature
-		
+			//skip segment signature

```

Comment:
Maybe we could add a call to `wrap_verifyimage` here, to verify the package contents too

Reply:
Done.